feature/expenses/src/main/java/com/serranoie/app/feature/expenses/AddExpenseScreen.kt

```
@@ -324,14 +330,27 @@ fun ExpenseBasicDetailsSection(
-            label = "Date",
-            titleHeader = false,
-            leadingIcon = Icons.Default.CalendarToday,
-            containerColor = null,
-            modifier = Modifier.fillMaxWidth()
+        DateTimeInput(
+            selectedDateTime = if (date.isNotEmpty()) {
+                try {
+                    val localDate =
+                        java.time.LocalDate.parse(date, DateTimeFormatter.ISO_LOCAL_DATE)
+                    java.util.Date.from(localDate.atStartOfDay(ZoneId.systemDefault()).toInstant())
+                } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)